Tests/QueuesFluentDriverTests/QueuesFluentDriverTests.swift

```diff
@@ -263,6 +348,18 @@ func XCTAssertThrowsErrorAsync<T>(
     }
 }
 
+func XCTAssertNoThrowAsync<T>(
+    _ expression: @autoclosure () async throws -> T,
+    _ message: @autoclosure () -> String = "",
+    file: StaticString = #filePath, line: UInt = #line
+) async {
+    do {
+        _ = try await expression()
+    } catch {
+        XCTAssertNoThrow(try { throw error }(), message(), file: file, line: line)
```

Comment:
I understand why, but this is a wat moment for sure 😆

Reply:
Well once I can update this to Swift 6 and just use Swift Testing, this nonsense will go away 🙃